graph/path/yen_ksp.go

```diff
@@ -0,0 +1,130 @@
+// Copyright ©2018 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package path
+
+import (
+	"sort"
+
+	"gonum.org/v1/gonum/graph"
+	"gonum.org/v1/gonum/graph/internal/set"
+)
+
+type yenShortest struct {
```

Comment:
This appears to be dead. Please remove it.

Reply:
No, yenShortest is still being used to sort the found shortest in order.

Reply:
Ah, sorry yes. Can you move it below the main function when you add the `sort.Interface` implementation. Normally we introduce things after they are used rather than before.